pruner/example_programs/fabric/fabric.p4

```diff
@@ -59,13 +108,14 @@ control FabricIngress (inout parsed_headers_t hdr,
     Next() next;
     IngressSliceTcClassifier() slice_tc_classifier;
     IngressQos() qos;
+
 #ifdef WITH_PORT_COUNTER
     PortCountersControl() port_counters_control;
 #endif // WITH_PORT_COUNTER
 #ifdef WITH_SPGW
     SpgwIngress() spgw;
 #endif // WITH_SPGW
-
+/*
```

Comment:
What is this comment doing here? Please remove it, we should keep as much from the original program as possible.